docs/current/api/990050-index.md

```diff
@@ -0,0 +1,5 @@
+---
+slug: /api
+---
+
+# Dagger API
```

Comment:
I want to understand this terminology better in our specific context.
- Should this be "Dagger Engine" instead?
- What's the difference between the API and the Engine?
- Is the CLI an interface to the API, to the Engine, or both?